Changelog.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## 0.9.0 RC
+## 0.9.0 RC 2
```

Comment:
The change log seems to have no mention of RC1 which is inconsistent with how 0.8.0-rc1 and 0.8.0-rc2 were included in the change log.